src/App.jsx

```diff
@@ -1,6 +1,8 @@
-import React from 'react';
+import React, { useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
+
+import { Tabs } from './components/Tabs/Tabs';
```

Comment:
```suggestion
import { Tabs } from './components/Tabs';
```
do not delete (restore) index.js file inside components/Tabs folder to make it possible (good practice)

Or (even better in this case - there is only one file), remove Tabs folder - place Tabs.js directly in components folder